plugins/woocommerce/client/blocks/assets/js/blocks/add-to-cart-with-options/variation-selector/frontend.ts

```
@@ -36,13 +38,91 @@ type Context = AddToCartWithOptionsStoreContext & {
+			const $el = $( e );
+			var name = '';
+			switch ( optionStyle ) {
+				case 'pills':
+					name = $el.attr( 'name' )
+					break;
+				case 'dropdown':
+					name = $el.data( 'wpContext' )['name'];
+					break;
+				default:
+					throw new Error( 'optionStyle not implemented!: ', optionStyle );
+					break;
```

Biome (2.1.2)

[error] This code will never be reached ...

... because this statement will throw an exception beforehand


(lint/correctness/noUnreachable)